tutorials/asr/ASR_with_NeMo.ipynb

```diff
@@ -0,0 +1,920 @@
+        "colab": {}
+      },
+      "source": [
+        "# --- Config Information ---#\n",
+        "from ruamel.yaml import YAML\n",
+        "config_path = './configs/config.yaml'\n",
+        "\n",
+        "yaml = YAML(typ='safe')\n",
+        "with open(config_path) as f:\n",
+        "    params = yaml.load(f)\n",
+        "print(params)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wUmq3p2Aw_5N",
+        "colab_type": "text"
+      },
+      "source": [
+        "NeMo models and modules can be used in any PyTorch code where torch.nn.Module is expected.\n",
+        "\n",
+        "However, NeMo's models are based on PytorchLightning's LightningModule and we recommend you use PytorchLightning for training and fine-tuning as it makes using mixed precision and distributed training very easy. So to start, let's create Trainer instance for training on GPU for 50 epochs"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "GUfR6tAK0k2u",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import pytorch_lightning as pl\n",
+        "trainer = pl.Trainer(gpus=1, max_epochs=50)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IEn2RyvgxxvO",
+        "colab_type": "text"
+      },
+      "source": [
+        "Next, we instantiate and ASR model based on our ``config.yaml`` file.\n",
+        "Note that this is a stage during which we also tell the model where our training and validation manifests are."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Cbf0fsMK09lk",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from omegaconf import DictConfig\n",
+        "params['model']['train_ds']['manifest_filepath']=train_manifest\n",
+        "params['model']['validation_ds']['manifest_filepath']=test_manifest\n",
+        "first_asr_model = nemo_asr.models.EncDecCTCModel(cfg=DictConfig(params['model']), trainer=trainer)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "inRJsnrz1psq",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Start training!!!\n",
+        "trainer.fit(first_asr_model)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jpYXX-GslR0E",
+        "colab_type": "text"
+      },
+      "source": [
+        "There we go! We've put together a full training pipeline for the model and trained it for 50 epochs. Let's start Tensorboard to see how training went. Note that WER stands for Word Error Rate and the lower it is, the better.\n",
```

Comment:
New section here as well to distinguish that this is separate from building the model. Maybe "Evaluating the Training with Tensorboard"? Not sure.

Reply:
added "Monitoring Training"